pkg/controller/etcdproxy/etcdproxy_controller.go

```diff
@@ -435,3 +455,118 @@ func (c *EtcdProxyController) handleObject(obj interface{}) {
 		return
 	}
 }
+
+// createEtcdProxyCACertConfigMap creates ConfigMap in controller namespace with Etcd Proxy CA certificate.
+func (c EtcdProxyController) createEtcdProxyCACertConfigMap(etcdstorage *etcdstoragev1alpha1.EtcdStorage) error {
+	// ConfigMap in controller namespace for the etcd proxy CA certificate.
+	caConfigMap, err := c.kubeclientset.CoreV1().ConfigMaps(c.config.ControllerNamespace).
+		Get(etcdProxyCAConfigMapName(etcdstorage), metav1.GetOptions{})
+	if errors.IsNotFound(err) {
+		data := map[string]string{"ca.pem": EtcdProxyCACert}
+		caConfigMap, err = c.kubeclientset.CoreV1().ConfigMaps(c.config.ControllerNamespace).
+			Create(newConfigMap(etcdstorage, etcdProxyCAConfigMapName(etcdstorage), c.config.ControllerNamespace, data))
+	}
+	if err != nil {
+		c.recorder.Event(etcdstorage, corev1.EventTypeWarning, ErrUnknown, err.Error())
+		return err
```

Comment:
this error most probably is not really saying much, more like "Conflict while update". I believe you have to give it some context.

Reply:
It gives the error message, as we're passing it with `err.Error()`. The error message contains information what happened and which resources are affected, including their name, namespace, version..

I think that's okay for now. But, I'm sure we can improve messages, and I'll create a follow-up PR.

Reply:
Created issue #41.